std/algorithm/iteration.d

```diff
@@ -5034,28 +5039,28 @@ if (isSomeChar!C)
         }
 
     public:
-        this(C[] s) pure @safe
+        this(Range s)
         {
-            import std.string : strip;
-            _s = s.strip();
+            import std.string : stripLeft;
+            _s = s.stripLeft();
```

Comment:
Note, I realized that strip isn't necessary -- the `getFirst` function is going to skip any trailing whitespace anyway.

Reply:
Likewise the `stripLeft` in `popFront`.

Reply:
Nevermind, you are correct.